app/services/classification/class_param.rb

```diff
@@ -1,41 +0,0 @@
-module Classification
```

Comment:
Kinda confused where did this code go exactly? Can you provide a bit of background?